qf_lib/data_providers/helpers.py

```diff
@@ -190,8 +190,8 @@ def tickers_dict_to_data_array(tickers_data_dict: Dict[Ticker, QFDataFrame],
     """
     # return empty xr.DataArray if there is no data to be converted
     requested_tickers, _ = convert_to_list(requested_tickers, Ticker)
-    requested_fields, _ = convert_to_list(requested_fields,
-                                          type(requested_fields) if not isinstance(requested_fields, Sequence) else str)
+    if not isinstance(requested_fields, Sequence) or isinstance(requested_fields, str):
+        requested_fields, _ = convert_to_list(requested_fields, type(requested_fields))
```

Comment:
I think we should still convert to list if requested_fields is of type str, because fields could be interpreted below as a list of chars

Reply:
We do - we have the 'or isinstance str' -> we convert to list